packages/ui/react-ui-editor/src/extensions/automerge/semaphore.ts

```diff
@@ -58,7 +65,7 @@ export class PatchSemaphore {
 
     // NOTE: null and undefined each come from automerge and repo respectively.
     if (newHeads === null || newHeads === undefined) {
-      // TODO: @alexjg this is the call that's resetting the editor state on click
+      // TODO(alexjg): this is the call that's resetting the editor state on click.
```

Comment:
noting that I'd likely interpret this as a todo _from_ alex rather than as a note to pass on to him